common/runtime/pom.xml

```diff
@@ -97,6 +119,15 @@
                     </annotationProcessorPaths>
                 </configuration>
             </plugin>
+<!--                    <plugin>-->
+<!--                        <artifactId>maven-surefire-plugin</artifactId>-->
+<!--                        <configuration>-->
+<!--                            <systemProperties>-->
+<!--                                <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>-->
+<!--                                <gcloud.test>true</gcloud.test>-->
+<!--                            </systemProperties>-->
+<!--                        </configuration>-->
+<!--                    </plugin>-->
```

Comment:
Why do you comment this? If it's not needed you can remove it but as far as I remember this is needed to have logs property formatted on tests

Reply:
Left over (sorry)

I initially added it, because I was thinking of some `@QuarkusTest`, but then realized that it's good enough to check the static-init race.